invenio_communities/models.py

```diff
@@ -293,12 +293,17 @@ def add_record(self, record):
         except:  # the record was not in the pending area
             pass
         key = current_app.config['COMMUNITIES_RECORD_KEY']
+        access_key = '_access'
         record.setdefault(key, [])
+        record.setdefault(access_key, {'read': []})
 
         assert self.id not in record[key]
         record[key].append(self.id)
         record[key] = sorted(record[key])
 
+        assert self.id not in record[access_key]['read']
```

Comment:
why the assert? do we really care? just add it if it is not already there ;)
we can even use a set here instead of an array(?)